testing/benches/all.rs

```diff
@@ -10,8 +10,9 @@ criterion_main!(benches);
 criterion_group!(benches, functions);
 
 fn functions(c: &mut Criterion) {
-    c.bench_function("Big table", |b| big_table(b, &100));
-    c.bench_function("Teams", |b| teams(b));
+    c.bench_function("Big table", |b| big_table(b, &100))
```

Comment:
Why did you use a chaining structure here rather than just adding a bench function with a separate function call?